src/language-js/utils.js

```diff
@@ -1320,6 +1320,10 @@ function isCallLikeExpression(node) {
   );
 }
 
+function isObjectProperty(node) {
+  return node.type === "ObjectProperty" || node.type === "Property";
+}
```

Comment:
`estree` [called it `Property`](https://github.com/estree/estree/blob/master/es5.md#property), better name it `isProperty`